Userland/Libraries/LibCore/ArgsParser.cpp

```diff
@@ -65,9 +65,9 @@ bool ArgsParser::parse(Span<StringView> arguments, FailureBehavior failure_behav
 
     for (size_t i = 0; i < m_options.size(); i++) {
         auto& opt = m_options[i];
-        if (opt.long_name) {
+        if (!opt.long_name.is_null()) {
```

Comment:
I think we're trying to actively avoid new users of the null state. `Optional<StringView>` instead?

Reply:
+1, please don't use the null state in new code.